### DISCUSSION TITLE
Issue with uploading large video file

### DISCUSSION THREAD

**Discussion by ylbeethoven** [immich-app/immich#8832] [Q&A] [answered]
Hi everyone,

I am trying to move my google photo assets to self-hosted immich. Everything works fine except I have two video files that can't be imported through the UI or immich-cli.

> Version: v1.101.0
> Platform: Docker on Linux VM
> Connection: Through tailscale

On the UI, I was getting api error as below.
![image](https://github.com/immich-app/immich/assets/1949110/70602db8-91ee-49d8-9425-8785af7ed5c2)

By using immich cli, the progress bar never moved. More importantly, it seems the cli double the size of the file
```bash
```bash
✘ ls -la
total 6.5G
drwxr-xr-x  5 xxx users  12K Apr 16 00:43 .
drwx------ 36 xxx users 4.0K Apr 16 10:21 ..
-rw-r--r--  1 xxx users 3.6G Apr 15 22:56 GOPR1598-069.MP4
-rw-r--r--  1 xxx users 2.9G Apr 15 22:56 GOPR1599-070.MP4
➜ immich upload GOPR1599-070.MP4
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | 0% | ETA: 0s | 0/6.2 GB: GOPR1599-070.MP4^C
~/Downloads took 17s
✘ immich upload GOPR1599-070.MP4 --dry-run
████████████████████████████████████████ | 100% | ETA: 0s | 6.2 GB/6.2 GB: GOPR1599-070.MP4
Would have uploaded 2 assets (6.2 GB)
```

Does anyone know if this is expected? Is there a size limit and how do I increase it temporarily?

Thank you.

**Comment by mmomjian** [comment, marked as answer]
I saw you are using Tailscale. Are you also using a reverse proxy server and/or Cloudflare to serve Immich?

**Reply by ylbeethoven** [reply]
ah, thanks for your help. I forgot I was using the cloudflare domain on the browser.

Accessing it directly using tailscale on the browser work. The size limit was probably imposed at the Cloudflare edge.

It is still weird the CLI double the file size and progress bar did not move when it was connected to the tailscale endpoint directly.

```bash
➜ cat ~/.config/immich/auth.yml
instanceUrl: http://home-album:2283/api
```

**Reply by mmomjian** [reply]
6.2 GB is probably just the total size of both files together.

Not sure about the CLI, that's strange.
